int32 id
float32 confidence
float32 distance_m
int32 x_min
int32 y_min
int32 x_max
int32 y_max
std_msgs/Header header
